Cards/CardBase.gd

```diff
@@ -84,6 +83,7 @@ func _handle_effects_queue(caster: Entity, base_target: Entity) -> void:
 			push_error("Did not trigger a hit on all targets for effect " + card_effect.resource_path)
 	
 	_card_effects_queue.remove_at(0)
+	_targets_triggered_hits.clear()
```

Comment:
this still might be a problem later on if we decide we want an effect that applies the same effect multiple times (for whatever reason), depending on how we do it. We can probably make it work still if the effect when triggered just files the list of effects with the repetition of the desired effect
I will approve and keep this in mind